### DISCUSSION TITLE
v2.5.0 broke ie11

### DISCUSSION THREAD

**Discussion by theChaosCoder** [livewire/livewire#3160] [7. Bugs (archived)] [answered]
IE11 is not working since v2.5.0. It complains something about Alpine V3...

**Comment by joshhanley** [comment]
@theChaosCoder there was a bug in 2.5.0, so please upgrade to 2.5.1. That should resolve the issue 🙂

Hope this helps!

**Reply by theChaosCoder** [reply]
v2.5.1 also does not work. I just tested it again, just to be sure...

console shows SCRIPT1002 syntax error
`window.addEventListener('alpine:initializing', () => {`

No problems with v2.4.4

**Reply by joshhanley** [reply]
@theChaosCoder can you screenshot the whole console error with the stack trace?

**Comment by joshhanley** [comment, marked as answer]
@theChaosCoder ah, so IE11 doesn't support short arrow functions, but one has been used here, which is in a script tag that doesn't get built and polyfilled.

https://github.com/livewire/livewire/blob/06f7c7681aaa447ddaa9c26c8a27cf8b8826c15c/src/LivewireManager.php#L294

That is a simple fix of submitting a PR to change `() => {` to `function() {` if you are up for it.

Hope this helps!

**Reply by theChaosCoder** [reply]
Thx it works again, here's the PR https://github.com/livewire/livewire/pull/3173

**Reply by joshhanley** [reply]
No worries 🙂  thanks for submitting the PR!
